admin/src/css/theme.less

```diff
@@ -176,3 +176,7 @@ ul {
 .btn-bottom-spacing  {
   margin-bottom: 5px;
 }
+
+.inactive {
+  opacity: 0.5;
```

Comment:
This also applies the bottom border. Also it'd be good to remove the cursor styling on hover. Consider instead using...
```suggestion
  color: @inactive-color;
  cursor: inherit;
```

Reply:
Fixed.